typescript/packages/subsurface-viewer/src/SubsurfaceViewer.stories.tsx

```diff
@@ -16,12 +16,32 @@ import { ExtendedLayerProps, LayerPickInfo } from "./layers/utils/layerTools";
 import { WellsPickInfo } from "./layers/wells/wellsLayer";
 import { FeatureCollection } from "geojson";
 import { ViewFooter } from "./components/ViewFooter";
+import { styled } from "@mui/material/styles";
 
 export default {
     component: SubsurfaceViewer,
     title: "SubsurfaceViewer",
 } as ComponentMeta<typeof SubsurfaceViewer>;
 
+const PREFIX = "Default";
```

Comment:
This looks like boilerplate `codemod` autogenerated code copied from elsewhere. I think there's room for improvement. At least I don't think the `PREFIX` is needed.

Reply:
you're right my mistake :-)